Gui/opensim/installer_files/Linux/opensim.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Version=1.0
+Type=Application
+Name=OpenSim
+Comment=OpenSim is an open source software for neuromusculoskeletal modeling, simulation and analysis.
```

Comment:
Consider this revision:

Neuromusculoskeletal modeling, simulation, and analysis.